src/builtin/builtin_echo.c

```diff
@@ -0,0 +1,41 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   builtin_echo.c                                     :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: r3dc4t <marvin@42.fr>                      +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/11/02 00:39:25 by r3dc4t            #+#    #+#             */
+/*   Updated: 2022/11/13 19:17:39 by al7aro           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "builtin.h"
+#include <stdio.h>
+
+static void	print_string(char **tab, t_bool nl)
+{
+	int	i;
+
+	i = 0;
+	while (*(tab + i))
+		printf("%s", *(tab + i++));
+	if (nl)
+		printf("\n");
+}
+
+t_error_code	builtin_echo(t_shell_op *sp, t_cmd *c)
```

Comment:
ok we need to talk about all the prints of the builtins
they can't use printf since we have pipes
they will need to write and read from fds that represent stdin & stdout